core/src/consensus/mod.rs

```diff
@@ -266,7 +266,10 @@ pub trait ConsensusEngine: Sync + Send {
         header.hash()
     }
 
-    fn can_change_canon_chain(&self, _header: &HeaderView) -> bool {
+    /// In PoW consensus, the higher scored block became the best block.
```

Comment:
became -> becomes?

Reply:
You're right. I'll change it to `becomes`.